src/Test.FunctionalTests.Sarif/Multitool/ValidateCommandTests.cs

```diff
@@ -47,6 +47,22 @@
         public void JSON1002_DeserializationError()
             => RunTest("JSON1002.DeserializationError.sarif");
 
+        [Fact]
+        public void JSON1006_ValueTooLarge_LocationId_Valid()
```

Comment:
>JSON1006_ValueTooLarge_LocationId_Valid [](http://example.com/codeflow?start=20&length=39)

This is too much test infrastructure for the work that you are doing. Are we going to have hundreds or thousands of test files in order to test every possible numeric value in the format? 

Please just create at most two test files, one for reach rule.

Please don't qualify your test name by Location id, is this the only numeric value we are testing? If so, why are we only testing the specific corner case we have found here?

As always, please don't just focus on the specific issue that's been flagged. Please don't just accept the code design patterns as you've found them and make things more complicated.

Instead, look for efficient analysis and testing that comprehensively addresses the problem that's been found.

Reply:
Please just create at most two test files, one for each rule.
----- yes, I created a file for valid and a file for invalid, and in each file there can be more than 1 case, I will merge the valid and invalid to 1 file.

Please don't qualify your test name by Location id, is this the only numeric value we are testing?  ---- It is, I only added the max for Location Id in the Definition file, so I can only test for it. 
I should also look for other fields those have similar issue and add the max attribute,
**But I see your other comments, is the way I am adding the max attribute the right way to go? Need some guidance here.**